relayer/socket/server.go

```diff
@@ -308,7 +308,7 @@ func (s *Server) parseEvent(msg *Request) *Response {
 			}
 			balance, err := chain.Provider.QueryBalance(ctx, req.Address)
 			if err != nil {
-				return response.SetError(err)
+				balance = types.NewCoin("", 0, 0)
```

Comment:
Should we ignore the error? I think we should log the error at appropriate place, and also propagate.

Reply:
Yeah.

Because empty denom and value denotes the error.